### PR TITLE
https://issues.redhat.com/browse/ACM-8855

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-8855

### CODE REVIEW

**Review by birsanv — CHANGES_REQUESTED** [review]
sam comment as for https://github.com/stolostron/rhacm-docs/pull/5949

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -89,7 +89,7 @@ kind: MyResource
     cluster.open-cluster-management.io/backup: cluster-activation
 ----
 
-*Note:* Regardless of the `cluster.open-cluster-management.io/backup` label value, secrets and config maps are restored when passive resources are restored. Since the `cluster-activation` value is ignored for these types of resources, set the `cluster.open-cluster-management.io/backup: cluster-activation` value to the resources using these secrets. This way, the resources are not restored until the activation step.
+*Note:* Regardless of the `cluster.open-cluster-management.io/backup` label value, secrets and config maps are restored when passive resources are restored. Since the `cluster-activation` value is ignored in this case, set the `cluster.open-cluster-management.io/backup: cluster-activation` value to the resources using these secrets and config map instead. This way, they are not restored until the activation step.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: looks good !

I think **config map instead** should be plural, but all else looks good

**Review by birsanv — APPROVED** [review]
lgtm , with the last comment

**Review by birsanv — APPROVED** [review]
lgtm

**Review by oafischer — APPROVED** [review]

**Pull request merged**